### PR TITLE
Add central log mgt event handler

### PR DESCRIPTION
**Pull request by AnuradhaSK**
### Proposed changes in this pull request
Add a central logs management module.

Part of Fix https://github.com/wso2/product-is/issues/5037

- [x] Depends on https://github.com/wso2/carbon-kernel/pull/3029
- [x] Kernel Release

TODO

- [x] When merging, Update the versions in the pom files to the latest framework version.

### CODE REVIEW

File: components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/hanlder/CentralLogger.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+/**
+ * Central log event handler for audit, and diagnostic logs.
+ */
+public class CentralLogger extends AbstractEventHandler {
+
+    @Override
+    public boolean isEnabled(MessageContext messageContext) {
+
+        if (super.isEnabled(messageContext)) {
+            return !isLegacyAuditLogsDisabled();
```
Comment (emswbandara) [inline components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/hanlder/CentralLogger.java:42]: this validation should be applied only for the audit logs in handleEvent method.

File: components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/internal/CentralLogMgtServiceComponent.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            log.debug("IdentityEventService set in Central logger.");
+        }
+    }
+
+    protected void unsetIdentityEventService(IdentityEventService identityEventService) {
+
+        CentralLogMgtServiceComponentHolder.getInstance().setIdentityEventService(null);
+        if (log.isDebugEnabled()) {
+            log.debug("IdentityEventService set in Central logger.");
+        }
```
Comment (mevan-karu) [inline components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/internal/CentralLogMgtServiceComponent.java:92]: Do we need this debug log?

File: components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/hanlder/CentralLogger.java (hunk 1 above)
Comment (AnuradhaSK) [reply components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/hanlder/CentralLogger.java]: addressed in 3687bdc

File: components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/internal/CentralLogMgtServiceComponent.java (hunk 2 above)
Comment (AnuradhaSK) [reply components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/internal/CentralLogMgtServiceComponent.java]: Addressed in f4b3622

**Pull request merged**